### PR TITLE
Adding some debug logs to help detail why frames are skipped

### PR DESCRIPTION
**Pull request by MSFT-Heba**
Adding some debug logs to help detail why frames are skipped

### CODE REVIEW

File: Source/RulesEngine/ServiceCallItem.cs [hunk 1]
```diff
@@ -320,12 +321,14 @@ public static ServiceCallItem FromFiddlerFrame(UInt32 frameId, ZipArchiveEntry c
                     // CONNECT Frames should not be in the analysis.
                     if (firstLineSplit[0] == "CONNECT")
                     {
+                        System.Diagnostics.Debug.WriteLine("CONNECT Frames should not be in the analysis.");
```
Comment (jasonsandlin) [inline Source/RulesEngine/ServiceCallItem.cs:324]: i think fiddler normally captures connect frames as well.  Try with a normally decrypted saz and ensure this doesn't result in logs that look like something went wrong

File: Source/RulesEngine/Utils.cs [hunk 2]
```diff
@@ -102,11 +102,85 @@ public static int GetFrameNumber(String filename)
 
         public static bool IsAnalyzedService(WebHeaderCollection headers, String customUserAgent)
         {
-            return ((headers["x-xbl-api-build-version"] != null && headers["x-xbl-api-build-version"].Equals("adk", StringComparison.OrdinalIgnoreCase))
-                || (headers["User-Agent"]!= null && headers["User-Agent"].Contains("XboxServicesAPI") && headers["x-xbl-client-name"] == null)
-                || headers["X-XBL-Build-Version"] != null  // XCE CS1.0 event
-                || headers["X-AuthXToken"] != null  // UTC upload CS2.1 events
-                || (!String.IsNullOrEmpty(customUserAgent) && headers["User-Agent"] == customUserAgent));
+            var result = false;
```
Comment (jasonsandlin) [inline Source/RulesEngine/Utils.cs:105]: what do the logs look like with a normal run?  just make sure it doesn't look like something went wrong

**Review by jasonsandlin — APPROVED** [review]

File: Source/RulesEngine/Utils.cs (hunk 2 above)
Comment (rakhad) [inline Source/RulesEngine/Utils.cs:105]: >var result = false; [](http://example.com/codeflow?start=12&length=19)

To make sure these aren't writing to the command line, just to the VS debug output?

**Review by rakhad — APPROVED** [review]
:shipit:

File: Source/RulesEngine/ServiceCallItem.cs (hunk 1 above)
Comment (MSFT-Heba) [reply Source/RulesEngine/ServiceCallItem.cs:324]: Yeah I already did that. We needed to log these a couple times while debugging things for ATG, hence why it's added here.

File: Source/RulesEngine/Utils.cs (hunk 2 above)
Comment (MSFT-Heba) [reply Source/RulesEngine/Utils.cs:105]: Normally, what happens is something along the lines of:

Analyzing Frame# 75
Analyzing https://userpresence.xboxlive.com/users/xuid(XUID)/devices/current/titles/current
header[x-xbl-api-build-version] is null
header[User-Agent] does not contain XboxServicesAPI
header[X-XBL-Build-Version] is null
header[X-AuthXToken] is null
customUserAgent is null or empty
Skipping

Analyzing Frame# 76
Analyzing https://achievements.xboxlive.com/users/xuid(XUID)/achievements?titleId=TITLEID&orderBy=title
header[x-xbl-api-build-version] is null
header[User-Agent] contains XboxServicesAPI and header[x-xbl-client-name] is null
header[X-XBL-Build-Version] is null
header[X-AuthXToken] is null
customUserAgent is null or empty
Processing Call

Analyzing Frame# 77
CONNECT Frames should not be in the analysis.
Skipping

Comment (MSFT-Heba) [reply Source/RulesEngine/Utils.cs:105]: So basically it explains what the headers contained and why it was either processed or skipped

Comment (MSFT-Heba) [reply Source/RulesEngine/Utils.cs:105]: Yeah just the VS debug output. I didn't want to corrupt the command line with too much information. Agreed?

Comment (rakhad) [reply Source/RulesEngine/Utils.cs:105]: I agree - we shouldn't make any changes to command line output without an additional command line argument.

Comment (jasonsandlin) [reply Source/RulesEngine/Utils.cs:105]: Sounds good

**Pull request merged**